### PR TITLE
Apply code formatters `black` & `isort` to code base

### PR DESCRIPTION
**Pull request by Nicoretti**
**_:memo: Note: Except for the changelog entry the entire changeset was automatically generated by python tooling._**

### CODE REVIEW

**Comment by littleK0i** [comment]
RIP commit history and git blame.

**Comment by Nicoretti** [comment]
Hi @littleK0i,

Thank you for your feedback. The change is inevitable because there is an ongoing effort to harmonize Exasol’s projects to simplify maintenance etc.. Reformatting the code base will consequently be unavoidable. 

Having said that:

I'll add a `.git-blame-ignore-revs` file. I need to merge this commit first, though, because Exasol has a `squash-commit` policy, so I can't add it in the same PR (SHA will change). For details on **_ignoring revs_**, visit https://git-scm.com/docs/git-blame or for a brief summary, visit https://graphite.dev/guides/git-blame-ignore-revs. https://github.com/orgs/community/discussions/5033

Best, 
Nico

File: pyexasol/formatter.py [hunk 1]
```diff
@@ -102,19 +111,24 @@ def escape_like(cls, val):
             val: Value to be escaped.
 
         Returns:
-            A string where all double quotes ``\\`` have been replaced 
-            with ``\\\\``, where ``%`` have been replaced with ``\%``,
-            where ``_`` have been replaced with ``\_``.
+            A string where all double quotes ``\\`` have been replaced
+            with ``\\\\``, where ``%`` have been replaced with ``\\%``,
+            where ``_`` have been replaced with ``\\_``.
```
Comment (tomuben) [inline pyexasol/formatter.py:116]: Are those additional backslashes intentional?

Comment (tomuben) [reply pyexasol/formatter.py:116]: Looks correct in documentation.

**Review by tomuben — APPROVED** [review]

**Pull request merged**